cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -689,7 +692,13 @@ def is_email_to_send_callback(self, status, time_original_request):
                 logger.info("duration_query > timeout_threshold_email %s", duration_query > timeout_threshold_email)
                 logger.info("email_sending_timeout and duration_query > timeout_threshold_email %s", email_sending_timeout and duration_query > timeout_threshold_email)
 
-                return email_sending_timeout and duration_query > timeout_threshold_email
+                number_done_emails_ok = True
+                if os.path.exists(self.scratch_dir + '/email_history'):
+                    done_email_files = glob.glob(self.scratch_dir + '/email_history/email_done_*.email')
+                    if len(done_email_files) >= 1:
+                        logger.info("number of done emails sent: %s", len(done_email_files))
```

Comment:
we should issue a warning here, to sentry. this is somehow anomalous, and we need to be aware